Spatial_searching/include/CGAL/Fuzzy_sphere.h

```diff
@@ -87,12 +87,12 @@ class Fuzzy_sphere_impl
     typename SearchTraits::Construct_cartesian_const_iterator_d construct_it=
         traits.construct_cartesian_const_iterator_d_object();
     typename SearchTraits::Cartesian_const_iterator_d cit = construct_it(c),
-        end = construct_it(c, 0);
+                                                      end = construct_it(c, 0);
     for (int i = 0; cit != end && (distance <= sq_inner_radius); ++cit, ++i) {
       if ((*cit) < rectangle.min_coord(i))
-        distance += (rectangle.min_coord(i)-(*cit))*(rectangle.min_coord(i)-(*cit));
+        distance += CGAL::square(rectangle.min_coord(i)-(*cit));
       if ((*cit) > rectangle.max_coord(i))
```

Comment:
Not new code, but should this be `else if`?

Reply:
Seems safe, yes.

Thanks for the review!